src/submission/views.py

```diff
@@ -599,13 +565,15 @@ def delete_frozen_author(request, article_id, author_id):
                     "email": author.email
                 },
         )
-        return redirect(
-            reverse_with_next(
-                'submit_authors',
-                next_url,
-                kwargs={'article_id': article_id}
+        if next_url:
+            return redirect(next_url)
+        else:
+            return redirect(
+                reverse(
+                    'submit_authors',
+                    kwargs={'article_id': article_id}
+                )
```

Comment:
Just a question since I've seen this a couple times: why does this change need unwinding / what issue is it trying to address?